packages/common/src/environment.ts

```diff
@@ -95,7 +95,7 @@ export const currentEditorUrl = editorUrls[getCurrentEnv()];
 export const githubAppClientId = githubAppClientIds[getCurrentEnv()];
 export const environmentDisplayName = environmentDisplayNames[getCurrentEnv()];
 export const currentOfficeJsRawSnippetsBaseRepoUrl = `https://raw.githubusercontent.com/OfficeDev/office-js-snippets/${
-  getCurrentEnv() === 'cdn' ? 'deploy-prod' : 'deploy-beta'
+  getCurrentEnv() === 'cdn' ? 'prod' : 'master'
```

Comment:
Can we make the 'master' -> 'main' change in office-js-snippets first?

Reply:
As we discussed, let's make that branch name change when we're back next year so we're around to deal with any related changes in other repos, etc.